### DISCUSSION TITLE
recipes/breakfast/crunch-french-toast/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#240] [Comments]
# recipes/breakfast/crunch-french-toast/

A collection of my recipes.

https://nicholaswilde.io/recipes/breakfast/crunch-french-toast/

**Comment by nicholaswilde** [comment]
Tried with the TJ's caramel sauce instead of making own but the corn flakes were not crispy enough to properly coat and stick to the bread. Also, the full hour was not used to have the corn flakes and caramel cool. Should make the caramel from scratch and use the full hour for cooling next time.

Also, make a 1/2 recipe option.
